docs/appendices/release-notes/6.0.0.rst

```diff
@@ -91,6 +91,8 @@ Scalar and Aggregation Functions
 Performance and Resilience Improvements
 ---------------------------------------
 
+- Improved join order binding and cross-join elimination.
```

Comment:
Could we express a bit what the improvement is about? Otherwise there not much value in adding this entry.

Maybe something like this? not the best, maybe you have a better suggestion?
```suggestion
- Improved performance and consistency of building JOIN plans 
  by using the user-defined order of join pairs directly from the parser.
```

Reply:
What this pr fixes is that explicit joins are created first, and then the implicit  joins. How about:

```
- Improved execution for queries with mixed implicit and explicit joins. Joins are now always executed in the original order of the query.
```

Reply:
Great, thanks!